app/controllers/organizations/adopter_fosterer/adopter_applications_controller.rb

```diff
@@ -19,7 +19,7 @@ def create
         notice: t(".success", message: MessagesHelper.affirmations.sample)
 
       # mailer
-      @org_staff = User.organization_staff(@pet.organization_id)
+      @org_staff = User.staff
```

Comment:
Just for my sanity as I have not checked, the tenant is set in the mailer scope isn't it?